src/Providers/Stripe/Card/Service.php

```diff
@@ -45,8 +45,16 @@ public function find(string $uid): Card
     public function create(Token $token = null): Card
     {
         if ($token) {
-            /** @var SetupIntent $response */
-            $response = SetupIntent::retrieve($token->value);
+            switch ($token->type) {
+                case StripeToken::SETUP_INTENT:
+                    /** @var SetupIntent $response */
+                    $response = SetupIntent::retrieve($token->value);
+
+                    break;
+                case StripeToken::TOKEN:
+                default:
+                    return new Card($token->value);
```

Comment:
As discussed, not keen on this as it's not actually creating a card / doing anything.